nexus/src/app/rack.rs

```diff
@@ -90,8 +93,36 @@ impl super::Nexus {
             })
             .collect();
 
+        let certificates: Vec<_> = request
+            .certs
+            .into_iter()
+            .enumerate()
+            .map(|(i, c)| {
+                // One-indexing for user-visible values.
```

Comment:
```suggestion
                // The indexes that appear in user-visible names for these certificates start from one (e.g., certificate names "default-1", "default-2", etc.).
```

(I initially misread this comment to mean that this index needed to match up with some 1-indexed thing that we exposed to end users, so I suggested this edit to be more explicit.)

Reply:
Done fb340d7f971b072951da271923156bf58fa313dc